lib/EnsEMBL/REST/Model/ga4gh/Beacon.pm

```diff
@@ -514,6 +505,13 @@ sub variant_exists {
   my $ref_allele = $input_coords->{'ref_allele'};
   my $alt_allele = $input_coords->{'alt_allele'};
 
+  # If alt allele is missing then we have a range query
+  # return any variant within start and end
+  my $range_query = 0;
+  if(!$alt_allele) {
+    $range_query = 1;
+  }
```

Comment:
I think that if the user optionally gives `variantType` to a [bracket query](http://docs.genomebeacons.org/variant-queries/#beacon-bracket-queries), `$range_query` will be 0. Is this intended?

Reply:
Yes, that is intended.